index.html

```diff
@@ -4,83 +4,219 @@
+		</div>
+		<div class="container value">
+			<div class="row">
+				<div class="col-lg-6 order-md-2 order-sm-1">
+					<img class="demo" src="./images/sanbox.png" width="400">
+					<!--<a href='https://www.freepik.com/psd/woman'>Woman psd created by freepik - www.freepik.com</a>-->
+				</div>
+				<div class="col-lg-6  order-md-1 order-sm-2">
+					<h1 class="pop">The XR sandbox for your creativity</h1>
+					<h3 class="motto">powered by the community-built 3D assets, music, animation and effects library</h3>
+
+				</div>
+
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row">
 				<div class="col-lg-6">
-					<video playsinline="" muted="" class="img-rounded video" src="images/demo.mp4" style="width:inherit" loop autoplay=""></video>
+					<img class="demo" src="./images/alldevice2.png" width="400">
 				</div>
+				<div class="col-lg-6">
+					<h1 class="pop">Share and enjoy many XR creations together in any device</h1>
+					<h3 class="motto">remove the barrier between your XR arts and your audience</h3>
 
+				</div>
 			</div>
+		</div>
+		<div class="end rounded shadow-lg">
+			<div class="container value mt-5">
+				<div class="row justify-content-center">
+					<div class="col-lg-9 center-close mt-5">
+						<h1 class="pop">Bring more people to contribute to the future of XR</h1>
+						<h3 class="motto">Our strength lies in the hands of our users, forming a mobile XR community welcoming everyone of all levels and backgrounds to share their creative story through art and music.
+						</h3>
+					</div>
+					<div class="col-lg-12">
+						<img style="width:inherit" src="./images/community.png">
+							<!--3D hands created by ThreeDee (https://www.handz.design/)-->
+					</div>
 
+				</div>
+			</div>
 		</div>
-		<div class="footer">
-			<div class="container">
-				© 2021 Virtision. All Rights Reserved.
 
-			</div><!-- /.navbar-collapse -->
+		<div class="container value mt-5">
+			<div class="row justify-content-center">
+				<div class="col-lg-12 center">
+					<h1 class="pop"> Be the first to get the app </h1>
+				</div>
+				<div class="col-lg-4">
+					<form class="form-inline validate" role="form" action="https://virtision.us1.list-manage.com/subscribe/post?u=9ab8c2f01a3f5aa677094fc1e&amp;id=4c5085e68c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form"
+						target="_blank" novalidate>
+						<div class="row justify-content-center">
+							<div class="form-group">
+								<input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
+								<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+							</div>
+						<button type="submit" class="btn btn-email btn-fill mt-4">NOTIFY ME</button>
+
+						</div>
+					</form>
+				</div>
 
+			</div>
 		</div>
+
+
+
 	</div>
+
+
+	<div class="footer">
+		<div class="container">
```

Comment:
it'd be nice if this had a background so it wouldn't just transparently go right over the body text
 
 but it's a super minor thing and not really that important right now